### PR TITLE
Documenting the webp support

### PR DESCRIPTION
**Pull request by abdelhady**
Based on this merged pull-request https://github.com/avalanche123/Imagine/pull/504 , webp is now supported, so let's please show this in docs ;)

### CODE REVIEW

File: docs/usage/introduction.rst [hunk 1]
```diff
@@ -170,7 +170,7 @@ The following example opens a Jpg image and saves it as Png format :
 Three options groups are currently supported : quality, resolution and flatten.
 
 .. TIP::
-   Default values are 75 for Jpeg quality, 7 for Png compression level and 72 dpi for x/y-resolution.
+   Default values are 75 for Jpeg quality, 7 for Png compression level, 75 for webp quality and 72 dpi for x/y-resolution.
```
Comment (mlocati) [inline docs/usage/introduction.rst]: Is it correct that WebP has a default quality of 75? Where did you find this value?

File: docs/usage/introduction.rst [hunk 2]
```diff
@@ -186,6 +186,7 @@ The following example demonstrates the basic quality settings.
    $imagine->open('/path/to/image.jpg')
       ->save('/path/to/image.jpg', array('jpeg_quality' => 50)) // from 0 to 100
       ->save('/path/to/image.png', array('png_compression_level' => 9)); // from 0 to 9
+      ->save('/path/to/image.jpg', array('webp_quality' => 50)) // from 0 to 100
```
Comment (mlocati) [inline docs/usage/introduction.rst]: What about using `.webp` instead of `.jpg`?

Comment (abdelhady) [reply docs/usage/introduction.rst]: It was silly of me :)

File: docs/usage/introduction.rst (hunk 1 above)
Comment (abdelhady) [reply docs/usage/introduction.rst]: The following lines are used to set the webp image quality:
```javascript
if (isset($options['webp_quality']) && in_array($format, array('webp'))) {
      $image->setImageCompressionQuality($options['webp_quality']);
}
```

and as per the [official documentation for `setImageCompressionQuality`](http://php.net/manual/en/gmagick.setcompressionquality.php), its `$quality`'s default value is 75

File: docs/usage/introduction.rst [hunk 3]
```diff
@@ -186,6 +186,7 @@ The following example demonstrates the basic quality settings.
    $imagine->open('/path/to/image.jpg')
       ->save('/path/to/image.jpg', array('jpeg_quality' => 50)) // from 0 to 100
       ->save('/path/to/image.png', array('png_compression_level' => 9)); // from 0 to 9
+      ->save('/path/to/image.webp', array('webp_quality' => 50)) // from 0 to 100
```
Comment (abdelhady) [inline docs/usage/introduction.rst]: @mlocati By the way, I don't know how changing this string would break Travis build !

**Pull request merged**